yarn.lock

```diff
@@ -2560,6 +2560,13 @@
   resolved "https://registry.yarnpkg.com/@types/color-name/-/color-name-1.1.1.tgz#1c1261bbeaa10a8055bbc5d8ab84b7b2afc846a0"
   integrity sha512-rr+OQyAjxze7GgWrSaJwydHStIhHq2lvY3BOC2Mj7KnzI7XK0Uw1TOOdI9lDoajEbSWLiYgoo4f1R51erQfhPQ==
 
+"@types/fb-watchman@2.0.4":
+  version "2.0.4"
+  resolved "https://packages.atlassian.com/api/npm/npm-remote/@types/fb-watchman/-/fb-watchman-2.0.4.tgz#3a02379aa90d86ceb965a2beaebd7e79faa81e82"
```

Comment:
This needs to be regenerated with the yarnpkg registry. I normally use `yarn install --registry https://registry.yarnpkg.com/`